spec/requests/sessions_spec.rb

```diff
@@ -1,32 +1,10 @@
 require 'rails_helper'
 
-RSpec.describe "Sessions", type: :request do
-  describe "GET /new" do
-    it "returns http success" do
-      get "/sessions/new"
+RSpec.describe 'Sessions', type: :request do
+  describe 'GET /new' do
+    it 'returns http success' do
+      get login_path
       expect(response).to have_http_status(:success)
     end
   end
-
```

Comment:
Why are these all deleted ?

Reply:
Because they were auto generated by rails and weren't testing anything/were failing, and we had already written our own which were sufficient